third-party/CMakeLists.txt

```diff
@@ -204,18 +213,6 @@ if(WIN32)
   elseif(TARGET_ARCH STREQUAL "X86")
     set(TARGET_ARCH ia32)
   endif()
-
-  GetBinaryDep(TARGET winpty
```

Comment:
What's the point of building winpty instead of using the pre-built dll?

Reply:
Let nvim only depends windows OS dlls, and keep binary exe independ.

Any reaaon not do that.

Reply:
and winpty-adapter.exe is no valueble for nvim